### ISSUE TITLE
[System] Bundle MT with JRE

### ISSUE THREAD

**Issue by wolfgang-ch** [mytourbook/mytourbook#381]
Used hints from https://kivio.org/blog/entry/2020/07/09/jre-mit-eclipse-rcp-anwendung-bundeln

**Comment by wolfgang-ch** [comment]
An embeddet JRE is now working :-)

but there is some work left

* Create MT release also without a JRE, needs some pom.xml tweaks
* Reduce size of the JRE, it is very large compared with embedded JRE's in other products


![image](https://user-images.githubusercontent.com/1283445/147814572-3c74d991-3336-480f-b449-0406063ed047.png)

![image](https://user-images.githubusercontent.com/1283445/147814582-d571be9b-236b-4f4b-ba97-d8f4845c36da.png)

**Comment by wolfgang-ch** [comment]
@FJBDev @telemaxx 
Do you think it is necessary that the JRE is also bundeled for Linux, this would be an additional zip file. Now it's implemented only for Win as setup file without the zip archive.

I think, Linux users are more the advanced users compared with Win users and do not need it :-?

![image](https://user-images.githubusercontent.com/1283445/147934023-f3d4764f-a137-4e1f-a194-786e8e3343a0.png)

**Comment by FJBDev** [comment]
@wolfgang-ch 
I agree, I don't think it's necessary to bundle up Java with the Linux installation because Linux users tend to be more advanced and if they are using Linux, I think they know how to install Java 😄

**Comment by telemaxx** [comment]
i think also, an additional bundled windows version is enough.

several times user reporting that mt is not starting since an update.
often the reason was an old java version on windows.

@wolfgang-ch : the bundled is not that much bigger, have you stripped down the embedded JRE?
i have expected a much bigger file.

**Comment by wolfgang-ch** [comment]
After I finished the implementation, I found this https://www.eclipse.org/justj/?page=documentation which should embed a jre easier. So I tried to replace my implementation with this but it failed. JustJ should also have the ability to optimize the JRE but after investing several days to get a working solution, I gave up with JustJ

The downloaded JRE zip file is much smaller than the embedded but for now I'm happy that it's working.

![image](https://user-images.githubusercontent.com/1283445/148016274-0ba12513-6d55-4646-b77e-21b9d0c217dc.png)
